src/accelerate/utils/modeling.py

```diff
@@ -1465,6 +1663,17 @@ def infer_auto_device_map(
                 f"offload_buffers=True."
             )
 
+    if device_minimum_assignment_memory:
+        devices_info = "\n".join(
+            f"  - {device}: {mem} bytes required" for device, mem in device_minimum_assignment_memory.items()
+        )
+        warnings.warn(
+            f"Based on the current allocation process, no modules could be assigned to the following devices due to "
+            f"insufficient memory:\n"
+            f"{devices_info}\n"
+            f"These minimum requirements are specific to this allocation attempt and may vary. Consider increasing "
+            f"the available memory for these devices to at least the specified minimum, or adjusting the model config."
+        )
```

Comment:
Can we use logger.info ? I prefer not to have too much verbose for this specific function as this is wildly used in transformers and users don't really have ways to fix it. Most of our users do device_map = "auto", so they can't really raise the available memory or adjust the model config.

Reply:
Thanks for pointing that out! I've addressed this in the latest commit.